### PR TITLE
[WC-905] Add documentation for Tooltip widget

### PR DESCRIPTION
**Pull request by diego-antonelli**

### CODE REVIEW

File: content/appstore/widgets/tooltip.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+title: "Tooltip"
+category: "Widgets"
+description: "Describes the Tooltip widget, which is available in the Mendix Marketplace."
+tags: ["marketplace", "marketplace component", "widget", "tooltip", "platform support"]
+#If moving or renaming this doc file, implement a temporary redirect and let the respective team know they should update the URL in the product. See Mapping to Products for more details.
+---
+
+## 1 Introduction
+
+[//]: # (TODO: Add real marketplace link after publish.)
```
Comment (ConnorLand) [inline content/appstore/widgets/tooltip.md:11]: When will the real marketplace link be live @diego-antonelli ? Ideally its nice to have it in the doc before publish.

**Review by Keraito — APPROVED** [review]
nothing to point out from technical perspective

**Review by raffiar — APPROVED** [review]

**Review by raffiar — APPROVED** [review]
It describes everything which  Tooltip has!

File: content/appstore/widgets/tooltip.md (hunk 1 above)
Comment (diego-antonelli) [reply content/appstore/widgets/tooltip.md:11]: We need the docs to publish apparently, so right after they approve.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**